x/did/internal/types/events.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+const (
+	EventTypeAddDidDoc     = "create_did_doc"
+	EventTypeAddCredential = "credential"
```

Comment:
```suggestion
	EventTypeAddCredential = "add_credential"
```

Reply:
code has updated from EventTypeAddCredential = "add_credential" TO EventTypeAddCredential = "add_credential"